render/tests/test-attributes.js

```diff
@@ -376,15 +376,15 @@ o.spec("attributes", function() {
 
 			o(a.dom.value).equals("1")
 		})
-		o("null becomes the empty string", function() {
+		o("null becomes 'null'", function() {
```

Comment:
Why is this behavior desirable? I can't think of a time where you'd want to set `"null"` as an attribute value.

Reply:
Just matching our behavior to the [HTML](https://html.spec.whatwg.org/multipage/form-elements.html#dom-option-value) and [WebIDL](https://heycam.github.io/webidl/#es-DOMString) specs. (Notably, the DOM's `option.value` does *not* have the `[TreatNullAs=EmptyString]` extended attribute.)

Reply:
I filed #1978 (but closed it for reasons not related to this).

Reply:
If you feel strongly enough about it, I can fix it pretty quickly.

(FYI, the mock still needed fixed, though.)

Reply:
Actually this is in conflict with #1865 which I'm currently trying to rebase. I had set up on treating `null` and `undefined` in vdom indiscriminately as a lack of attribute, see https://github.com/MithrilJS/mithril.js/pull/1865/commits/fa53a66ab775c9661c594bb24b9baf6fd1528974#diff-0c7e3694f9865906ffe0847957a6c3a9

I'll bring it up in gitter as well for visibility.